tool/src/test/java/org/wildfly/security/tool/CredentialStoreCommandTest.java

```diff
@@ -316,6 +316,7 @@ public void testGenerateKeyPairDSA() throws CredentialStoreException {
 
     @Test
     public void testGenerateKeyPairECDSA() throws CredentialStoreException {
+        Assume.assumeFalse(IS_IBM); //Don't run if using IBM JDK https://issues.redhat.com/browse/ELY-2070
```

Comment:
@Ashpan Would you be able to create a new ELY issue for unignoring these tests on IBM JDK and then reference that ELY issue in the comment here.

Reply:
Done! https://issues.redhat.com/browse/ELY-2214